src/control/zoom/ZoomControl.cpp

```diff
@@ -184,10 +195,9 @@ auto ZoomControl::getScrollPositionAfterZoom() const -> utl::Point<double> {
         return {-1, -1};
     }
 
-    return (this->scrollPosition * this->zoom) - this->zoomWidgetPos + this->unscaledPixels;
+    return this->scrollPosition * this->zoom - this->zoomWidgetPos + unscaledPixels;
```

Comment:
For consistency, it might be nice to change `unscaledPixels` to `this->unscaledPixels`...